frontends/main/src/app-pages/CertificatePage/DigitalCredentialDialog.tsx

```diff
@@ -0,0 +1,212 @@
+import React from "react"
+import Image from "next/image"
+import { Dialog, Typography, styled } from "ol-components"
+import { ButtonLink } from "@mitodl/smoot-design"
+import VerifyIcon from "@/public/images/icons/verify.svg"
+
+const Content = styled.div({
+  display: "flex",
+  flexDirection: "column",
+  gap: "16px",
+})
+
+const InfoPanel = styled.div(({ theme }) => ({
+  backgroundColor: theme.custom.colors.lightGray1,
+  padding: "16px",
+  marginTop: "28px",
+  borderRadius: "8px",
+  boxShadow:
+    "0 3px 8px 0 rgba(37, 38, 43, 0.12), 0 2px 4px 0 rgba(37, 38, 43, 0.10)",
+}))
+
+const Info = styled.dl({
```

Comment:
<img width="1278" height="893" alt="image" src="https://github.com/user-attachments/assets/be3cde45-fe13-454c-9da8-57cf15f7deb6" />

I think there's an issue with overflow with this container. I had AI generate my test data for my credentials to randomize it a bit, and it inserted some longer strings for issued to and criteria. You can see in this screenshot how they overflow outside the right bounds of the container. An `overflow: "auto"` here will add a scrollbar, but we might want to be more explicit about width to have the text properly wrap.